pyincore_data/utils/nsiutil.py

```diff
@@ -89,21 +89,186 @@ def assign_hazus_specific_structure_type(
+                f_arch.append(10)
+                w_arch.append(10)
+            elif full_occ_type_ == "EDU1" and no_stories_ > 1:
+                f_arch.append(11)
+                w_arch.append(9)
+            elif full_occ_type_ == "EDU2" and no_stories_ > 1:
+                f_arch.append(11)
+                w_arch.append(10)
+            elif full_occ_type_ == "GOV1":
+                f_arch.append(14)
+                w_arch.append(19)
+            elif full_occ_type_ == "GOV2":
+                f_arch.append(14)
+                w_arch.append(11)
+            elif full_occ_type_ == "IND1":
+                f_arch.append(9)
+                w_arch.append(8)
+            elif (
+                full_occ_type_ == "IND2"
+                or occ_type_ == "IND3"
+                or occ_type_ == "IND4"
+                or occ_type_ == "IND5"
+                or occ_type_ == "IND6"
+            ):
+                f_arch.append(9)
+                w_arch.append(7)
+            elif full_occ_type_ == "REL1":
+                f_arch.append(13)
+                w_arch.append(13)
+            elif (
+                full_occ_type_ in RES_TYPES
+                and (found_type_ == "B" or found_type_ == "S")
+                and no_stories_ <= 1
+            ):
+                f_arch.append(2)
+                w_arch.append(1)
+            elif (
+                full_occ_type_ in RES_TYPES
+                and (found_type_ == "B" or found_type_ == "S")
+                and no_stories_ > 1
+            ):
+                f_arch.append(4)
+                w_arch.append(1)
+            elif (
+                full_occ_type_ in RES_TYPES
+                and (
+                    found_type_ == "C"
+                    or found_type_ == "P"
+                    or found_type_ == "I"
+                    or found_type_ == "W"
+                    or found_type_ == "F"
+                )
+                and no_stories_ <= 1
+            ):
+                f_arch.append(1)
+                w_arch.append(1)
+            elif (
+                full_occ_type_ in RES_TYPES
+                and (
+                    found_type_ == "C"
+                    or found_type_ == "P"
+                    or found_type_ == "I"
+                    or found_type_ == "W"
+                    or found_type_ == "F"
+                )
+                and no_stories_ > 1
+            ):
+                f_arch.append(3)
+                w_arch.append(1)
+            else:
+                print("Did not match hurricane archetype mappings")
+                print(occ_type_)
+                f_arch.append(0)
+                w_arch.append(0)
+
+            # Update wind archetypes based on building area
+            if (
+                w_arch[len(w_arch) - 1] == "1"
+                and no_stories_ <= 1
+                and area_sqft >= 1550
+            ):
+                w_arch[len(w_arch) - 1] = "3"
```

Comment:
Same here, no quotes, this should just be w_arch[len(w_arch) - 1] = 3

Reply:
good catch, thank you, updated